marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/EntitySearchManager.java

```diff
@@ -96,6 +97,14 @@ public StringHandle search(SearchQuery searchQuery) {
             String query = queryDef.serialize();
             StructureWriteHandle handle = new StringHandle(buildSearchOptions(query, searchQuery)).withMimetype("application/xml");
             RawCombinedQueryDefinition rcQueryDef = queryMgr.newRawCombinedQueryDefinition(handle, queryDef.getOptionsName());
+
+            // If an entity has been selected, then apply this transform
+            List<String> entityTypeIds = searchQuery.getQuery().getEntityTypeIds();
```

Comment:
Is it possible to select more than one entity type? If not, can we change this to a string instead of a List<String>? We don't want the code to imply that something is possible when it's not.

Reply:
@rjrudin When AllEntities are selected, there can be more than 1 entityTypeId

Reply:
@rjrudin Yes Rob. When AllEntities are selected there can be more than 1 entityTypeId